package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mongoose-elasticsearch-xp",
-  "version": "5.2.0",
+  "version": "0.0.0-semantically-released",
```

Comment:
should not it be updated to 5.2.0 ?

Reply:
Nope. It should be in such form.

Semantic-release calculates new version such way:
- take last published version from npm
- try to find a `tag` in github repo with such version
- take commit messages after founded `tag`
- look at commit prefixes and decide which version to publish major/minor/fix
- after that it get npm version from first step and add proper numbers

So `version` value from `package.json` does not use and should contain some informative notice. In our case `0.0.0-semantically-released`.

Reply:
Just run `semantic-release-cli setup` and i take care of rest problems if they occur. ;)